packer/rocky/variables.pkrvars.hcl.example

```diff
@@ -0,0 +1,33 @@
+# Packer Variables File (Optional)
+# This file allows you to set default values for variables
+# You can create a copy of this file as `variables.auto.pkrvars.hcl`
+# and Packer will automatically load it.
+
+# GCP Project Configuration
+project_id = "scality-prod-ga-runners"
```

Comment:
Add generic project name

Reply:
I must insist that no reference to the gcp project is made here, this is a public open source repo and is used by others. Remove all reference to `scality-prod-ga-runners` from the code.